.npmrc

```diff
@@ -0,0 +1 @@
+@brown-ccv:registry=https://npm.pkg.github.com
```

Comment:
I think you want this to read `@thepolicylab`

Reply:
so it should be `@thepolicylab:registry`?